book/src/building/swift-packages/README.md

```diff
@@ -238,3 +233,38 @@ cd SwiftProject
 swift run
 # You should see "Hello from Rust!" in your terminal.
 ```
+
+# Usage on Linux
+Usage on Linux is a little bit different than on macOS. This guide will help you set up SwiftPM and Cargo. The general workflow is:
```

Comment:
Let's explicitly mention the ways that it is different.

So, up here instead of saying it's "a little bit different" just summarize the main way(s) that it is different.

Then your guide below walks the user through addressing those differences.